scripts/build-image-with-chef.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 DIR_NAME="packer-$(basename -s .json $1)"
 IMAGE_NAME="$(grep vm_name $1 | awk '{print $2}' | sed -e 's/\"//g' | sed -e 's/,//g')"
+IP1="nameserver 140.211.166.130"
+IP2="nameserver 140.211.166.131"
```

Comment:
IMO this should not be set here but instead be variables in the packer template. Take a look at how ``chef_version`` and ``mirror`` is used. Also the variable name should be more descriptive.